andela/travel_tool_front
# #160710842 Implement edit guest house centre functionality

#### What does this PR do?
 This PR ensures that a travel admin can edit an existing guest house
 
#### Description of Task to be completed?
- Write test for the  edit guest house endpoint
- Write code for the edit guest house functionality. 

#### How should this be manually tested?
- Clone the repository
- Run `make start` docker build command on the terminal to install all dependencies and run the App in the docker development environment server.
- On the browser, type the localhost URL ` http://localhost:3000` and Login with your Andela email account into the App.
- On the Requests page, click on the `Residence` link on the side navigation bar and click on `Manage` and the accommodation page  should be displayed.
- Create a new guest house center or click on one from the available list of guest house centers available on the page.
- Clicking on a guest house center should open a detail page showing details of the guest house center.
- On the guest house details page, click on the `Edit Guest House` link, which opens a modal form.
- Edit the guest house center data accordingly and click on `save changes` button.
- The guest house center data is updated as expected and displayed on the guest house details page

 #### Any background context you want to provide?
- Make sure that you are logged in as a `Super admin` or `Travel admin` in order for you to access the `Accommodation page`

#### What are the relevant pivotal tracker stories?
- [160710842](https://www.pivotaltracker.com/story/show/160710842)

#### Screenshots (if appropriate)

Screenshot showing edit guest house modal form

<img width="1277" alt="screen shot 2018-10-07 at 16 55 46" src="https://user-images.githubusercontent.com/12033494/46582588-041df000-ca52-11e8-9940-7b968dcbed6f.png">



#### Questions:
- None
